jdbc/src/main/java/site/ycsb/db/JdbcDBClient.java

```diff
@@ -184,8 +188,23 @@ public void init() throws DBException {
     String passwd = props.getProperty(CONNECTION_PASSWD, DEFAULT_PROP);
     String driver = props.getProperty(DRIVER_CLASS);
 
+  /* The SQL Syntax for Scan depends on the DB engine
```

Comment:
The if block for sqlserver recently moved down within our null check for the driver string. These should all be down there.


This is probably the merge conflict too.

Reply:
Ok I rebased and committed the merge.  I'm not sure if have something else to do on the PR or if that's ok.